### PR TITLE
Add black

### PR DESCRIPTION
**Pull request by aexvir**
https://github.com/ambv/black/

### CODE REVIEW

File: docker-black/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-alpine3.7
+
+WORKDIR /app
+RUN pip install black
```
Comment (chauffer) [inline docker-black/Dockerfile]: pin a version

File: docker-black/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+LABEL name=docker-black version=dev \
+      maintainer="Alex Viscreanu <aexvir@kiwi.com>"
+
+ENTRYPOINT [ "black" ]
```
Comment (chauffer) [inline docker-black/Dockerfile]: Any specific reason for using `ENTRYPOINT`? This is usually more hassle than anything else, I'd do `CMD` here.

Comment (aexvir) [reply docker-black/Dockerfile]: In order to allow both refactoring and only checking differences black should be launched with different arguments. That's why.

File: docker-black/Dockerfile (hunk 1 above)
Comment (aexvir) [reply docker-black/Dockerfile]: Right now it's on alpha stage but ok, I'll pin it to 18.4a2, which is the one I'm using right now.

**Comment by underyx** [comment]
https://github.com/kiwicom/dockerfiles/pull/12/commits/cf9b22e1b4a66518a464797317584f940f6dcbbf should probably be only in #16

**Pull request closed without merging**